lottie-swift/src/Public/MacOS/BundleImageProvider.swift

```diff
@@ -34,11 +34,17 @@ public class BundleImageProvider: AnimationImageProvider {
   
   public func imageForAsset(asset: ImageAsset) -> CGImage? {
     
-    if asset.name.hasPrefix("data:"),
-      let url = URL(string: asset.name),
-      let data = try? Data(contentsOf: url),
-      let image = NSImage(data: data) {
-      return image.CGImage
+    if asset.name.hasPrefix("data:") {
+        let encodedAsset = asset.name
+        let mimeLookupUpperBound = 25
+        let searchRange = encodedAsset.startIndex..<encodedAsset.index(encodedAsset.startIndex, offsetBy: mimeLookupUpperBound)
+        let mimeRange = encodedAsset.range(of: ";base64,", range: searchRange)
+        let base64String = String(encodedAsset[mimeRange!.upperBound..<encodedAsset.endIndex])
```

Comment:
It probably isn't safe to assume that any asset with `data:` also contains `;base64,` -- can we handle this gracefully instead of force-unwrapping?